.github/actions/display-jest-test-summary/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Display Jest Test Summary
+
+inputs:
+  test_report_path:
+    type: string
+    description: Path of test report HTML file
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Display Jest Test Summary
+      uses: actions/github-script@v6
+      env:
+        TEST_PATH: ${{ inputs.test_report_path }}
+      with:
+        script: |
+          const fs = require('fs');
```

Comment:
@JoshuaLicense @AnnaDodson  any thoughts on this approach?

Reply:
@sdh100shaun I have rewritten the action